libcodechecker/analyze/analysis_manager.py

```diff
@@ -153,6 +153,9 @@ def __eliminate_argument(arg_vect, opt_string, has_arg=False):
         # Build out custom invocation for dependency generation.
         command = [command[0], '-E', '-M', '-MT', '__dummy'] + command[1:]
 
+        # Remove empty arguments
+        command = [i for i in command if len(i) >= 1]
```

Comment:
> *From PEP 8, in the “Programming Recommendations” section:*
> For sequences, (strings, lists, tuples), use the fact that empty sequences are false.

```python
command = [e for e in command if e]
```
(using `e` as in **e**lem, you are not iterating indices here.)

Also, the comment above is missing the `.` at the end of the sentence.